core/src/test/scala/org/apache/spark/sql/IssueTestSuite.scala

```diff
@@ -17,6 +17,32 @@ package org.apache.spark.sql
 
 class IssueTestSuite extends BaseTiSparkSuite {
 
+  // https://github.com/pingcap/tispark/issues/255
+  test("Group by with first") {
+    ti.tidbMapDatabase("tpch_test")
+    val q1 = spark.sql("""
```

Comment:
may be better to change this test to `run(sql)` form.

Reply:
Done